bindings/python/test/environment/gravitational/earth/__init__.py

```diff
@@ -0,0 +1,19 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Physics
+# @file           bindings/python/test/environment/gravitational/earth/__init__.py
+# @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+import pytest
+
+import ostk.physics as physics
+
+################################################################################################################################################################
+
+
+
+
+################################################################################################################################################################
```

Comment:
```suggestion
```

Reply:
Why was this resolved? I don't think we should keep this.

Reply:
@apaletta3